src/controllers/orderController.js

```diff
@@ -57,11 +57,36 @@ const createOrder = async (req, res) => {
   res
     .status(StatusCodes.CREATED)
     .json({ clientSecret: paymentIntent.client_secret, order });
+  /*
+  #swagger.summary = 'Create a new order and process payment'
+  #swagger.description = 'Creates a new order and processes payment using Stripe.'
+  #swagger.tags = ['Orders']
+  #swagger.parameters['body'] = {
+    in: 'body',
+    description: 'Order information including order items, subtotal, tax, and total',
+    required: true,
+    schema: {
+      $ref: '#/definitions/Order' 
+    },
+  }
+  #swagger.responses[201] = {
+    description: 'Order created successfully',
+    schema: { $ref: '#/definitions/Order' } 
```

Comment:
The endpoint returns an object like this: `{ "order": < the order object we put in the definitions block in swagger.js> }`. We can make sure the docs match by matching that shape here:

```suggestion
    schema: { order: { $ref: '#/definitions/Order' } }
```